factcast-store/src/main/resources/db/changelog/factcast/issue1865/fix_index_for_enum.sql

```diff
@@ -0,0 +1,3 @@
+-- fix the index index_for_enum from issue170
+drop index if exists index_for_enum;
+create index index_for_enum on fact using btree((header->>'ns'),(header->>'type'));
```

Comment:
the new index needs to be created concurrently in order not to lock the fact table